engine/src/main/java/org/terasology/rendering/opengl/FrameBuffersManager.java

```diff
@@ -252,109 +344,123 @@ public void saveScreenshot() {
             }
         };
 
-        CoreRegistry.get(ThreadManager.class).submitTask("Write screenshot", task);
-
+        threadManager.submitTask("Write screenshot", task);
```

Comment:
This line originally fetched the threadManager from the CoreRegistry and used it inline. This generated a warning as the result of the CoreRegistry.get() method could return null. The warning is no longer there but the possibility of a null threadManager remains. How defensively should code here? Should I add a check and log an error if the threadManager is null after all?